extensions/wikia/AdEngine/js/lookup/rubiconVulcan.js

```diff
@@ -0,0 +1,125 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.rubiconVulcan', [
+	'ext.wikia.adEngine.lookup.lookupFactory',
+	'wikia.document',
+	'wikia.log',
+	'wikia.window'
+], function (factory, doc, log, win) {
+	'use strict';
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Use of JavaScript strict mode may result in unexpected behaviour in some browsers. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AStrictMode)